clean.sh

```diff
@@ -0,0 +1,3 @@
+for fixture in ./test/fixtures/*; do
+  rm -rf ./test/fixtures/$fixture/node_modules;
```

Comment:
I think something like this would be more robust:
```
git clean -xfd test/fixtures
```

Reply:
This also won't accidentally delete checked-in fixture `node_modules` (such as in the `transpile-node-modules` fixture).

Reply:
I think that is less robust, as it removes all files rather than just the ones we want to remove. Also, that would remove untracked files that someone might be working on. For example, if you added a new fixture and ran tests, those fixtures would be removed. We could potentially use the `-X` flag but I also don't want to worry about different versions of `git` on various machines, etc.

Reply:
Good point about untracked files. Using`-X` definitely makes sense.

Is different git versions actually a concern? Even git `1.6.0`, (released in 2008) [supports all those flags](https://git-scm.com/docs/git-clean/1.6.0). Support probably goes even further back, so it doesn't seem like different git versions is a concern.

Also I think using git for this rather than shell scripts arguably makes it more portable (i.e. windows).

Reply:
Good point about portability, sometimes I forget windows exists lol. I will update to use the -X flag